src/components/row.js

```diff
@@ -20,11 +26,46 @@ function Row(props) {
   const handleReveal = () => {
     props.handleReveal(props.state.state);
   };
-
+  
   const sort = (aDistricts) => {
-    return aDistricts;
+    sortedDistricts = {};
+    if(aDistricts) {
+      Object.keys(aDistricts).sort((district1, district2) => {
+        const sortColumn = sortData.sortColumn;
+        let value1 = aDistricts[district1][sortColumn];
+        let value2 = aDistricts[district2][sortColumn];
+        if (sortColumn === 'district') {
+          value1 = district1;
+          value2 = district2;
+        } else {
+          value1 = parseInt(value1);
+          value2 = parseInt(value2);
+        }
+
+        if (sortData.isAscending) {
+          return value1 > value2 ? 1 : (value1 == value2) && district1 > district2 ? 1 : -1;
```

Comment:
You don't need to repeat the same logic again two lines down. You can simply muliply with -1 if `sortData.isAscending` is `false`